src/main/java/com/crowdin/cli/commands/picocli/DownloadTranslationsSubcommand.java

```diff
@@ -0,0 +1,85 @@
+package com.crowdin.cli.commands.picocli;
+
+import com.crowdin.cli.client.ProjectClient;
+import com.crowdin.cli.commands.Actions;
+import com.crowdin.cli.commands.NewAction;
+import com.crowdin.cli.commands.functionality.FsFiles;
+import com.crowdin.cli.properties.ParamsWithFiles;
+import com.crowdin.cli.properties.PropertiesWithFiles;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Command(
+    name = CommandNames.TRANSLATIONS,
+    sortOptions = false
+)
+public class DownloadTranslationsSubcommand extends ActCommandWithFiles {
+
+    @CommandLine.Option(names = {"-b", "--branch"}, paramLabel = "...", order = -2)
+    protected String branchName;
+
+    @CommandLine.Option(names = {"--ignore-match"}, order = -2)
```

Comment:
Please specify description keys explicitly for all parameters. For the `download` command, it works automatically based on the message keys from the resource file. Since this command has a different name, it can't get these descriptions.

![Screenshot 2024-10-30 at 09 43 25](https://github.com/user-attachments/assets/d0631af1-f428-4169-9d1b-dcda34d8f0c4)